src/main/java/frc/robot/subsystems/PoseEstimatorSubsystem.java

```diff
@@ -61,11 +74,15 @@ public void periodic() {
     }
 
     public void updateVisionMeasurementTimestamp() {
-        timeStamp = Timer.getFPGATimestamp() - (Robot.LIMELIGHT_SUBSYSTEM.getTl() / 1000) - (Robot.LIMELIGHT_SUBSYSTEM.getCl() / 1000);
+        _timeStamp = Timer.getFPGATimestamp() - (Robot.LIMELIGHT_SUBSYSTEM_ONE.getTl() / 1000) - (Robot.LIMELIGHT_SUBSYSTEM_ONE.getCl() / 1000);
+    }
+
+    public void updateVisionMeasurmentTimestampSecondLimelight() {
+        _timeStamp2 = Timer.getFPGATimestamp() - (Robot.LIMELIGHT_SUBSYSTEM_TWO.getTl() / 1000) - (Robot.LIMELIGHT_SUBSYSTEM_ONE.getCl() / 1000);
     }
     
     public void resetOdometryPoseToLimelight() {
-        resetOdometryPose(Robot.DRIVE_TRAIN_SUBSYSTEM.getGyroscopeRotation(), Robot.DRIVE_TRAIN_SUBSYSTEM.getSwerveModulePositions(), Robot.LIMELIGHT_SUBSYSTEM.getGyroBasedRobotPose());
+        resetOdometryPose(Robot.DRIVE_TRAIN_SUBSYSTEM.getGyroscopeRotation(), Robot.DRIVE_TRAIN_SUBSYSTEM.getSwerveModulePositions(), Robot.LIMELIGHT_SUBSYSTEM_ONE.getGyroBasedRobotPose());
```

Comment:
add a parameter with type LimelightSubsystem, since that can be variable now that there are two.

Reply:
@matt-orlando I cannot make a instant command for a button bind to run "resetOdometryPose, can you check in VS code?

Reply:
@matt-orlando I tested everything in glass, it shows the live robot but not in the right location. Zielen the theory here is if we run "resetPosition" the pose estimator should correct to the right position, that's the status. I will use your recommendations tomorrow but I am honestly just trying to get this shit to work for auto driving you feel me?

Reply:
I did forget to remove some methods from when we had the whole code duplicate/ seprate classes, I will fix that soon

Reply:
Do you mean like `new Trigger().onTrue(new InstantCommand(() -> POSE_ESTIMATOR_SUBSYSTEM.resetOdometryPoseToLimelight()));`?

Reply:
My goal is to eliminate duplicate code, so when we do test it, we have less testing to do. If we do have to fix it, we only have to fix it in one place :)

Reply:
For a button on the controller it would look like `GAMEPAD.getButton(ButtonCode.X).onTrue(new InstantCommand(() -> POSE_ESTIMATOR_SUBSYSTEM.resetOdometryPoseToLimelight()));`